test/OpenTelemetry.Instrumentation.Http.Tests/OpenTelemetry.Instrumentation.Http.Tests.csproj

```diff
@@ -15,7 +15,6 @@
   </ItemGroup>
 
   <ItemGroup>
-    <PackageReference Include="System.Net.Http" Condition="'$(TargetFramework)' == 'net462'" />
```

Comment:
I'm going to approve this but FYI what I really want is for the tests to verify everything works if the user decides to take the NuGet bits. But we should also make sure it works with the direct reference. Probably need to run the tests through both modes to do it correctly? Going to take a bit of effort to pull that off, not sure if the juice is worth the squeeze.

Reply:
With this PR, the test is going to cover the net462 framework reference of `System.Net.Http`, do you think such test coverage can represent NuGet `System.Net.Http` (especially considering that the NuGet hasn't been updated since 2018) or we need both?



> what I really want is for the tests to verify everything works if the user decides to take the NuGet bits

Could you explain a bit more here? What's the main scenario and how big is the user base?

Reply:
It is definitely old, but I don't see anything on the NuGet saying it is deprecated and it has been downloaded 1.7**B** times so a lot of people are consuming it! The scenario is user takes a dependency on the NuGet and its bits are used instead of the shipped runtime assembly,

Reply:
#4915